diff --git a/src/main/java/com/book/dao/LendDao.java b/src/main/java/com/book/dao/LendDao.java
index f706a87..356ea98 100644
--- a/src/main/java/com/book/dao/LendDao.java
+++ b/src/main/java/com/book/dao/LendDao.java
@@ -88,4 +88,22 @@ public class LendDao {
         return list;
 
     }
+
+    public Lend getLend(long sernum) {
+        final Lend lend = new Lend();
+        jdbcTemplate.query("SELECT * FROM lend_list WHERE sernum = ?", new Object[]{sernum}, new RowCallbackHandler() {
+            public void processRow(ResultSet resultSet) throws SQLException {
+                lend.setBackDate(resultSet.getDate("back_date"));
+                lend.setBookId(resultSet.getLong("book_id"));
+                lend.setLendDate(resultSet.getDate("lend_date"));
+                lend.setReaderId(resultSet.getInt("reader_id"));
+                lend.setSernum(resultSet.getLong("sernum"));
+            }
+        });
+        return lend.getSernum() > 0 ? lend : null;
+    }
+
+    public int updateBackDate(long sernum, Date backDate) {
+        return jdbcTemplate.update("UPDATE lend_list SET back_date = ? WHERE sernum = ?", new Object[]{df.format(backDate), sernum});
+    }
 }
diff --git a/src/main/java/com/book/service/LendService.java b/src/main/java/com/book/service/LendService.java
index 2a271cb..0297094 100644
--- a/src/main/java/com/book/service/LendService.java
+++ b/src/main/java/com/book/service/LendService.java
@@ -31,4 +31,16 @@ public class LendService {
         return lendDao.myLendList(readerId);
     }
 
+    public boolean renewBook(long sernum) {
+        Lend lend = lendDao.getLend(sernum);
+        if (lend != null) {
+            java.util.Date backDate = lend.getBackDate();
+            if (backDate != null) {
+                backDate.setDate(backDate.getDate() + 30);
+                return lendDao.updateBackDate(sernum, backDate) > 0;
+            }
+        }
+        return false;
+    }
+
 }
