diff --git a/src/main/java/com/book/web/WebConfig.java b/src/main/java/com/book/web/WebConfig.java
index a122f5a..80844a3 100644
--- a/src/main/java/com/book/web/WebConfig.java
+++ b/src/main/java/com/book/web/WebConfig.java
@@ -1,11 +1,13 @@
 package com.book.web;
 
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.ComponentScan;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.web.servlet.ViewResolver;
 import org.springframework.web.servlet.config.annotation.DefaultServletHandlerConfigurer;
 import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+import org.springframework.web.servlet.config.annotation.InterceptorRegistry;
 import org.springframework.web.servlet.config.annotation.ResourceHandlerRegistry;
 import org.springframework.web.servlet.config.annotation.WebMvcConfigurerAdapter;
 import org.springframework.web.servlet.view.InternalResourceViewResolver;
@@ -16,6 +18,8 @@ import org.springframework.web.servlet.view.InternalResourceViewResolver;
 @ComponentScan("com.book.web")
 public class WebConfig extends WebMvcConfigurerAdapter {
 
+    @Autowired
+    private UserInterceptor userInterceptor;
 
     @Bean
     public ViewResolver viewResolver() {
@@ -30,6 +34,13 @@ public class WebConfig extends WebMvcConfigurerAdapter {
         configurer.enable();
     }
 
+    @Override
+    public void addInterceptors(InterceptorRegistry registry) {
+        registry.addInterceptor(userInterceptor)
+                .addPathPatterns("/**")
+                .excludePathPatterns("/img/**", "/js/**", "/css/**", "/html/**");
+    }
+
     @Override
     public void addResourceHandlers(ResourceHandlerRegistry registry) {
         registry.addResourceHandler("/img/**")
@@ -38,4 +49,4 @@ public class WebConfig extends WebMvcConfigurerAdapter {
         registry.addResourceHandler("/css/**").addResourceLocations("/static/css/");
         registry.addResourceHandler("/html/**").addResourceLocations("/static/html/");
     }
-}
\ No newline at end of file
+}
